assets/admin_themes/Sea_Green/breadcrumb/breadcrumb.scss

```diff
@@ -64,4 +64,7 @@ ol.breadcrumb.title-bar-breadcrumb {
 
 #breadcrumb-container {
     width: 100%;
+}
+ol.breadcrumb.title-bar-breadcrumb a:focus{
+    outline-offset: -2px !important
 }
```

Comment:
the scss files are not build so the change is not visible 
yarn install
gulp build_theme OR gulp watch_theme
gulpfile.js has also othere commands at the top if you need to build it
dont commit changes to nodemodules folder